ci-operator/step-registry/ipi/conf/ipi-conf-commands.sh

```diff
@@ -41,6 +41,16 @@ if [ -n "${BASELINE_CAPABILITY_SET}" ]; then
 capabilities:
   baselineCapabilitySet: ${BASELINE_CAPABILITY_SET}
 EOF
+        if [ -n "${ADDITIONAL_ENABLED_CAPABILITIES}" ]; then
```

Comment:
In some future world, folks may want to add caps to the default capability set.  We may want to allow for that with:

```sh
if [ -n "${BASELINE_CAPABILITY_SET}${ADDITIONAL_ENABLED_CAPABILITIES}" ]; then
	echo "Adding 'capabilities: ...' to install-config.yaml"
	printf 'capabilities:\n' >> "${out}"
	if [ -n "${BASELINE_CAPABILITY_SET}" ]; then
		printf '  baselineCapabilitySet: %s\n' "${BASELINE_CAPABILITY_SET}" >> "${out}"
	fi
	if [ -n "${ADDITIONAL_ENABLED_CAPABILITIES}" ]; then
		printf '  additionalEnabledCapabilities:\n' >> "${out}"
		for CAPABILITY in ${ADDITIONAL_ENABLED_CAPABILITIES}; do
			printf '  - %s\n' "${CAPABILITY}"
		done
	fi
fi
```